📦Update version to match npm

congphan3003 opened the pull request.

yagnaSoma commented on package.json:
```diff
@@ -95,6 +94,7 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@emotion/core": "^10.0.22",
+    "@snaphunt/snaphunt-ui": "^0.8.4",
```
Why is this moved to dependencies?
I can't see package-lock.json

yagnaSoma reviewed (changes requested):
Please update the redux-form-sui version and update the same on snaphunt-web

congphan3003 committed:
📦Update version to match npm


📦Update version to match npm


📦Update version to match npm


📦Update version to match npm
